scripts/qa_crowbarsetup.sh

```diff
@@ -437,6 +437,11 @@ function add_sdk_repo
     esac
 }
 
+function add_hcp_repo
+{
+    zypper ar -p 1 -f http://$susedownload/ibs/Devel:/Cloud:/6:/HPC/SLE_12_SP1/Devel:Cloud:6:HPC.repo
```

Comment:
please add a comment about the -p 1 like `needed for crowbar package overrides with possible version downgrades`

and better make it 92 to be between ptf (90) and default (99) so that we can test pull-requests using the normal UPDATEREPOS / PTF mechanism.